packages/frontend-vitals-monitoring/src/index.ts

```diff
@@ -42,13 +51,17 @@ export const frontendVitalsInit = (
   if (metrics.includes("avgResponseTime")) {
     calculateAvgResponseTime(server, staticPathsRegexp);
   }
+  if (metrics.includes("maxConcurrentConnections")) {
+    calculateMaxConcurrentConnections(server, interval);
```

Comment:
Sorry one tiny thing, we want to 'sample' the number of connections more frequently than we log the outputs. By passing in `interval` here we take a sample of the number of connections once every 10 seconds, and then log it immediately.

I think something like `1000` might be better than `interval` here. Getting the number of connections every second seems like a reasonable frequency as a starting point.

Reply:
Ok make sense, because when I was testing I did see some occurrences where it was returning 0 and I was thinking it might be because of the above ?